backend/src/LibExecution/Interpreter.fs

```diff
@@ -796,6 +796,18 @@ let execute (exeState : ExecutionState) (vm : VMState) : Ply<Dval> =
 
         | RaiseNRE nre -> raiseRTE (RTE.ParseTimeNameResolution nre)
 
+        | CheckIfFirstExprIsUnit reg ->
```

Comment:
Out of context, this DU case name isn't totally clear. Maybe just `RequireExprToReturnUnit` or something like that?

Reply:
It's probably fine - maybe a CLEANUP to think about this would be useful tho